src/js/components/ServiceSearchFilter.js

```diff
@@ -0,0 +1,65 @@
+import mixin from 'reactjs-mixin';
+/* eslint-disable no-unused-vars */
+import React from 'react';
+/* eslint-enable no-unused-vars */
```

Comment:
i actually didn't know that existed. i would 100% rather do that. A lot of our codebase is with the `/*` way unfortunately